package-res/resources/web/pentaho/data/_doc/IAbstractPropertyFilter.jsdoc

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright 2010 - 2015 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @ignore
+ * @classdesc The `IAbstractPropertyFilter` interface extends the `IAbstractFilter` interface
+ * to allow creation of types of filters for a data table.
+ *
```

Comment:
Not descriptive. The main characteristic of this class (and its descendants) is that it extracts a given property and compares its value to something (the `value`) using some comparison `operation`